tools/serverpod_cli/lib/src/generator/dart/library_generators/model_library_generator.dart

```diff
@@ -12,6 +12,14 @@ import 'package:serverpod_cli/src/generator/types.dart';
 import 'package:serverpod_serialization/serverpod_serialization.dart';
 import 'package:serverpod_service_client/serverpod_service_client.dart';
 
+/// The name of the method used to convert a model to JSON.
+const String _toJsonMethodName = 'toJson';
+
+/// The name of the method used to convert a model to JSON for protocol serialization.
+const String _toJsonForProtocolMethodName = 'toJsonForProtocol';
+
+const String _serverSideServerpodUrl = 'package:serverpod/serverpod.dart';
```

Comment:
Suggestion: Use `serverpodUrl` instead.

This is already available there with the parameter set to `true`.